src/url-replacements.js

```diff
@@ -95,6 +95,17 @@ class UrlReplacements {
       return documentInfoFor(this.win_).pageViewId;
     });
 
+    this.set_('QUERY_STRING', (param, defaultValue = "") => {
+      assert(param, 'The first argument to QUERY_STRING, the query string ' +
+          /*OK*/'param is required');
+      const url = parseUrl(this.win_.location.href);
+      const params = parseQueryString(url.search);
+
+      return (typeof params[param] !== "undefined") ?
```

Comment:
Please keep in mind that I am fairly new to node. When I tried to use hasOwnProperty I got an error that the params object doesn't have that method. It seems that JS has a different type for associative arrays vs instantiations of Object(). If I am missing something here please let me know but for now I couldn't make this change.